tutorials/frontend/react-apollo/app-final/README.md

```diff
@@ -7,14 +7,14 @@ Tutorial
 Tech stack
 ----------
 
-- Frontend
+## - Frontend
```

Comment:
Since you have added header, `-` can be removed at the front.

Reply:
** Thank you **

Reply:
Thank you

Reply:
Changes are made.